themes/Admin/js/admin.js

```diff
@@ -22,3 +22,21 @@ if (selectAllElement) {
         toggleSelectAll(e.target);
     });
 }
+
+
+// function for recycler get requests issued directly from select box
+function sendGetRequest(selectedValue) {
```

Comment:
The name of this function sounds like a generic function to send a get request, but it's tied specifically to the Recycler. Please rename to something more specific.

Reply:
Good point, I will change it to something more specific.